### ISSUE TITLE
What is general status of this library?

### ISSUE THREAD

**Issue by dzintars** [microsoft/redux-dynamic-modules#156]
Hi all. Curious about the general status of this library. It has not seen any updates for a while, so I am not sure is it abandoned or it is just pretty much done. Is it safe to dive into it?
Tnx.

**Comment by LFFATE** [comment]
It's definitely abandoned

**Comment by dzintars** [comment]
Then probably it would be marked as archived... ¯\_(ツ)_/¯

**Comment by owen26** [comment]
I've only recently started to dive into this library. It's unfortunate to see it hasn't been updated for quite a long time. I think both the design and use cases for this library is extremely good for splitting lazy-loaded react "modules", which I couldn't find any equivalent on the internet.

But I'm also a bit hesitant to introduce this library into my current project due to the fact this is a pretty low-level lib that requires your project to refactor in a way that follow the pattern that it is designed for. There is a cost to replace it again when it becomes no longer compatible with your other dependencies.

**Comment by dzintars** [comment]
Hi @fostyfost.
Thank you for pinging me.
To be honest right away, i would not use your library just because lack of your reputation. At least i don't see it. It probably sounds rude, but this is how i personally look at the libraries to include in the projects.
I really wish you luck with your project and really hope it will take off, because `redux-dynamic-modules` is really great but unfortunately not well known and thus seemingly abandoned feature/library.

**Comment by fostyfost** [comment]
@dzintars I totally agree with your opinion to choose only reliable libraries. I do the same. Thank you for your wishes =)
